src/watcher.js

```diff
@@ -23,7 +23,12 @@ const loadPredicates = {
         const lastReference = currentChatReference;
         const currentChat = twitch.getCurrentChat();
 
-        if (currentChat && currentChat === lastReference) return false;
+        if (currentChat && currentChat === lastReference) {
+            return false;
+        }
+        if (lastReference && currentChat.props.channelID.toString() === lastReference.props.channelID.toString()) {
```

Comment:
i actually want it to reload from our api in the case when a user navs to a new page from the same user.. emotes and bots can get updated